integration/options.go

```diff
@@ -57,6 +57,25 @@ const (
 
 	// defaultUseTChannelClientForTruncation determines whether we use the tchannel client for truncation by default.
 	defaultUseTChannelClientForTruncation = true
+
+	// defaultRepairThrottle determines the throttle per namespace repair by default.
+	defaultRepairThrottle = time.Duration(1 * time.Minute)
+
+	// defaultRepairTimeJitter determines the repair jitter by default.
+	defaultRepairTimeJitter = time.Duration(1 * time.Second)
+
+	// defaultRepairInterval determines the interval between repairs by default.
+	defaultRepairInterval = time.Duration(2 * time.Minute)
+
+	// defaultNumShards sets the default number of shards.
+	defaultNumShards = 1024
+
+	// defaultFetchSeriesBlocksBatchSize sets the number of series blocks to fetch in batch, by default.
+	defaultFetchSeriesBlocksBatchSize = 0
+
+	// defaultFetchSeriesBlocksBatchConcurrency sets the number of series blocks to fetch in
+	// batch concurrently, by default.
+	defaultFetchSeriesBlocksBatchConcurrency = 0
```

Comment:
What values do we set these if they are zero?  Or we just don't set them from the default?